### ISSUE TITLE
fight.direction got a very high VL value

### ISSUE THREAD

**Issue by qeterme** [NoCheatPlus/Issues#316]
We started testing our 1.10 server and we got 9223372036854775807 VL value for fight.direction and they're only playing with vanilla client without any game modifications.

---- Version information ----
#### Server ####
git-Spigot-7745d45-02d704b ~MC: 1.10.2~
  detected: 1.10.2
#### NoCheatPlus ####
Plugin: 3.15.0-SNAPSHOT-sMD5NET-b1037
MCAccess: 1.10 / Spigot-CB-1.10_R1
Features:
  blocks: BlocksMC1_4 | BlocksMC1_5 | BlocksMC1_6_1 | BlocksMC1_7_2 | BlocksMC1_8 | BlocksMC1_9 | BlocksMC1_10
  checks: FastConsume | Gutenberg | HotFixFallingBlockPortalEnter | AttackFrequency | FlyingFrequency | KeepAliveFrequency
  defaults: pvpKnockBackVelocity
  packet-listeners: UseEntityAdapter | MovingFlying | OutgoingPosition | KeepAliveAdapter | SoundDistance
Hooks: AllViolations~NCP~ 1.0 | MyPet 1.0
#### Related Plugins ####
ProtocolLib v4.1.0-SNAPSHOT-b308

We don't know how to reduce this. It happened only sometimes. Sometimes with shield, sometimes without that.

We have a full custom config. Linked here: https://gist.github.com/qeterme/d8ce18697e58bbc6f681ed94241ca412

We only have MyPet as hooks if this matter.

**Comment by MarkElf** [comment]
Duplicate of issue #276.
The VL is identical between both submitted issues; the issue is persistent when lowering the penalty based upon personal testing. Could the issue be tied with the introduction of the sweep attack as it doesn't exist prior to 1.9 (If I remember correctly, it was first brought to my attention ~Build 960)?

https://github.com/NoCheatPlus/NoCheatPlus/commit/51885cdc56c8104ad7308fa76bb37b206d6c3f10
https://github.com/NoCheatPlus/NoCheatPlus/commit/dfb65b23fb331ee4daf508136b6a63cf7dea1e9b

**Comment by asofold** [comment]
@MarkElf Thank you for relating to the other ticket, this is really helpful :).

The sweep attack stuff could be a candidate, if sweep attacks were not properly detected. Otherwise the code rather adds exemption for sweep attacks - the main point is the specific value of the violation, which leads to a preset value that should not have been passed as a violation level.

Consider fixed in build 1039.
